content/backend/graphql-python/2-queries.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Queries
+description: Creating Your First Queries
+---
+
+### About the Django structure
+It's important to understand how the Django project structure works before moving on.
+
+On the last chapter, you created a *Django Project*, which holds everything related to your application. However, Django separates the project into *apps*. Think about apps as a[separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns). You will have two apps, one for Users and one for the Links.
+
+Another essential decision is about the database. Django works out of the box with [SQLite](https://www.sqlite.org/) – a file based database – which you will use. On production systems, you should have a more robust database, such as [PostgreSQL](https://www.postgresql.org/). 
+
+### Creating the Links app
+
+<Instruction>
+
+On the project's root (next to the `manage.py` file), create the `links` app:
+
+```bash
```

Comment:
for all bash commands, it's also possible to add the directory where the command should be executed, the syntax is identical to the one for indicating the file that corresponds to a code block, e.g.:

``````
```bash(path=".../graphql-python/hackernews")
python manage.py startapp links
```
``````